Add arguments to dftFilter2D

take1014 opened the pull request:
resolves #10683

alalek commented:
Please add some tests for these using of these args (width != full_width) - tests are required for new functionality.

take1014 commented:
@alalek 
Thanks for the follow.
I added a test. Since I thought that "width" and "height" are not necessary as arguments of this function, I deleted them.

alalek commented:
Well done!
I have rebased patch on 3.4 branch and update test check condition (via "expected" values).

take1014 commented:
Thanks for the follow. 
I agree this change.Thank you very much.
